src/main/java/bowling/domain/Pins.java

```diff
@@ -0,0 +1,43 @@
+package bowling.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Pins {
+    private List<Pin> pins;
+
+    public Pins() {
+        this(PinGenerator.generate());
+    }
+
+    public Pins(List<Pin> pins) {
+        this.pins = pins;
+    }
+
+    public int knockDown(int knockedDownPinCount) {
+        List<Pin> pinList = pins.stream()
+                .filter(Pin::isStanding)
+                .limit(knockedDownPinCount)
+                .collect(Collectors.toList());
+
+        pinList.forEach(pin -> pin.knockDown());
```

Comment:
```suggestion
        pinList.forEach(Pin::knockDown);
```
조금 더 간단하게 표현도 가능 할 것 같습니다.

Reply:
아래커밋에 반영했습니다.
d1f678a